randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -643,24 +650,32 @@ Extra - room_id: [19]
   * L0 Hatch to Processing Access/Door to Sova Processing (Lower)
   * Extra - door_idx: (40,)
   > Beside Lava
-      Varia Suit and Wave Beam
+      All of the following:
+          Wave Beam
+          Any of the following:
+              Varia Suit
+              Damage Runs (Beginner) and Heat Damage ≥ 5
 
 > Door to Processing Access (Upper); Heals? False
   * Layers: default
   * L0 Hatch to Processing Access/Door to Sova Processing (Upper)
   * Extra - door_idx: (75,)
   > Beside Lava
-      Varia Suit
+      Varia Suit or Heat Damage ≥ 5
 
 > Door to Garbage Chute; Heals? False
   * Layers: default
   * L0 Hatch to Garbage Chute/Door to Sova Processing
   * Extra - door_idx: (78,)
   > Beside Lava
       All of the following:
-          Gravity Suit and Screw Attack
-          # Heat Run Logic
-          Varia Suit
+          Screw Attack
+          Any of the following:
+              # Heat Run logic (need Gravity to do Screw Attack in the lava)
+              # Gravity - Lava and Heat Damage
+              Gravity Suit and Damage Runs (Intermediate) and Lava Damage ≥ 110
+              # Varia and Gravity - No Damage
+              Gravity Suit and Varia Suit
```

Comment:
common requirement of gravity can be moved into `all of`
```
all of
 screw and grav
 any of
   damage runs
   varia
```

Reply:
Fixed